### ISSUE TITLE
Data18 wrong image

### ISSUE THREAD

**Issue by pibi73** [PAhelper/PhoenixAdult.bundle#1671]
With Data18 , lot's of video are loaded with wrong background and poster .

any idea?
Best regards

**Comment by diamondpete** [comment]
I just tested a random scene with no issues, so an example is needed. Please provide the info requested in #218 #4 when opening issues. Bear in mind that if Data18 puts the image on the scene page it gets scraped.

**Comment by chempke** [comment]
I have the same problem with many scenes. This private.com scene, for example, pulls images from a bangbros scene. 

1. Scene URL: [https://www.data18.com/scenes/3120614](https://www.data18.com/scenes/3120614) Sofia Lee in Alpine Lovers
3. File Name: data18scenes - 3120614 
4. Log File
```
2023-01-17 13:42:32,851 (7f64a27a0b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmRlczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo0Y2U5OWYxZWFhZDQwY2E4czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczIwCkRhdGExOHNjZW5lcyAzMTIwNjE0czQKbmFtZXMxODcKJTJGdm9sdW1lMSUyRk5BUyUyRjQlMkUlMjBWaWRlbyUyRjMlMkUlMjBYWFglMkYzJTJFJTIwQ29udGVudCUyRlByaXZhdGUlMkUyMDIyJTJFMTElMkUwNyUyRVNvZmlhJTJFTGVlJTJFRnVja2VkJTJFSW4lMkVUaGUlMkVTdW4lMkVYWFglMkUyMTYwcCUyRU1QNC1XUkIlMkZkYXRhMThzY2VuZXMlMjAtJTIwMzEyMDYxNCUyRW1wNHM4CmZpbGVuYW1lczQwCmE4NjQxN2M3MDI2NjU3Nzc5NTI1Y2U1NTEwY2ZmYjU1Y2I4YWIwODVzOApwbGV4SGFzaHM3CjE3Njg4MzJzOApkdXJhdGlvbnM2CjI4NjYxMXMyCmlkcjAK
2023-01-17 13:42:32,852 (7f64a27a0b38) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2023-01-17 13:42:32,854 (7f64a27a0b38) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmRlczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGU2CnMxNgo0Y2U5OWYxZWFhZDQwY2E4czE3Cm9wZW5TdWJ0aXRsZXNIYXNoczIwCkRhdGExOHNjZW5lcyAzMTIwNjE0czQKbmFtZXMxODcKJTJGdm9sdW1lMSUyRk5BUyUyRjQlMkUlMjBWaWRlbyUyRjMlMkUlMjBYWFglMkYzJTJFJTIwQ29udGVudCUyRlByaXZhdGUlMkUyMDIyJTJFMTElMkUwNyUyRVNvZmlhJTJFTGVlJTJFRnVja2VkJTJFSW4lMkVUaGUlMkVTdW4lMkVYWFglMkUyMTYwcCUyRU1QNC1XUkIlMkZkYXRhMThzY2VuZXMlMjAtJTIwMzEyMDYxNCUyRW1wNHM4CmZpbGVuYW1lczQwCmE4NjQxN2M3MDI2NjU3Nzc5NTI1Y2U1NTEwY2ZmYjU1Y2I4YWIwODVzOApwbGV4SGFzaHM3CjE3Njg4MzJzOApkdXJhdGlvbnM2CjI4NjYxMXMyCmlkcjAK
2023-01-17 13:42:32,854 (7f64a27a0b38) :  INFO (agentkit:961) - Searching for matches for {'openSubtitlesHash': '4ce99f1eaad40ca8', 'name': 'Data18scenes 3120614', 'filename': '%2Fvolume1%2FNAS%2F4%2E%20Video%2F3%2E%20XXX%2F3%2E%20Content%2FPrivate%2E2022%2E11%2E07%2ESofia%2ELee%2EFucked%2EIn%2EThe%2ESun%2EXXX%2E2160p%2EMP4-WRB%2Fdata18scenes%20-%203120614%2Emp4', 'plexHash': 'a86417c7026657779525ce5510cffb55cb8ab085', 'duration': '1768832', 'id': '286611'}
2023-01-17 13:42:32,855 (7f64a27a0b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/286611/tree'
2023-01-17 13:42:32,865 (7f64a27a0b38) :  INFO (__init__:68) - ***MEDIA TITLE [from media.name]*** Data18scenes 3120614
2023-01-17 13:42:32,865 (7f64a27a0b38) :  INFO (PAsearchSites:51) - mediaTitle w/ possible abbreviation: Data18scenes 3120614
2023-01-17 13:42:32,881 (7f64a27a0b38) :  INFO (PAsearchSites:59) - mediaTitle w/ possible abbrieviation fixed: Data18scenes 3120614
2023-01-17 13:42:32,893 (7f64a27a0b38) :  INFO (PAsearchSites:71) - ^^^^^^^ siteNum: 1071
2023-01-17 13:42:32,894 (7f64a27a0b38) :  INFO (PAsearchSites:72) - ^^^^^^^ Shortening Title
2023-01-17 13:42:32,897 (7f64a27a0b38) :  INFO (PAsearchSites:96) - Search Title (before date processing): 3120614
2023-01-17 13:42:32,898 (7f64a27a0b38) :  INFO (__init__:70) - {'siteName': 'data18scenes', 'searchDate': None, 'siteNum': 1071, 'searchTitle': '3120614'}
2023-01-17 13:42:32,898 (7f64a27a0b38) :  INFO (PAsearchData:23) - SearchData.title: 3120614
2023-01-17 13:42:32,899 (7f64a27a0b38) :  INFO (PAsearchData:29) - SearchData.filename: data18scenes - 3120614
2023-01-17 13:42:32,899 (7f64a27a0b38) :  INFO (__init__:107) - Provider: siteData18Scenes
2023-01-17 13:42:32,901 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/sys/search.php?index=&key=&key2=&next=1&page=0"
2023-01-17 13:42:33,125 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "ee35e0865e684208b018d50b98434ae8.gz"
2023-01-17 13:42:33,132 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/scenes/3120614"
2023-01-17 13:42:33,376 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "9892c782eee0473d884bba7488f9a9d4.gz"
2023-01-17 13:42:33,386 (7f64a27a0b38) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2023-01-17 13:42:33,386 (7f64a27a0b38) :  DEBUG (runtime:924) - Response: [200] str, 732 bytes
2023-01-17 13:42:34,467 (7f64a27a0b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZGVzNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyODY2MTFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzRXM2kyaHlySlRWZ0h2d2d2dDcyMXl6dGlraVp6ejdyNDRMMmdEVUZzbTJXYmVYZDdXanwxMDcxfD9sYW5nPWRlczQKZ3VpZHM1Nwo0VzNpMmh5ckpUVmdIdndndnQ3MjF5enRpa2laeno3cjQ0TDJnRFVGc20yV2JlWGQ3V2p8MTA3MXxzMgppZHIwCg__
2023-01-17 13:42:34,468 (7f64a27a0b38) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2023-01-17 13:42:34,470 (7f64a27a0b38) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZGVzNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgoyODY2MTFzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzOTkKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovLzRXM2kyaHlySlRWZ0h2d2d2dDcyMXl6dGlraVp6ejdyNDRMMmdEVUZzbTJXYmVYZDdXanwxMDcxfD9sYW5nPWRlczQKZ3VpZHM1Nwo0VzNpMmh5ckpUVmdIdndndnQ3MjF5enRpa2laeno3cjQ0TDJnRFVGc20yV2JlWGQ3V2p8MTA3MXxzMgppZHIwCg__
2023-01-17 13:42:34,470 (7f64a27a0b38) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://4W3i2hyrJTVgHvwgvt721yztikiZzz7r44L2gDUFsm2WbeXd7Wj|1071|?lang=de
2023-01-17 13:42:34,472 (7f64a27a0b38) :  DEBUG (model:234) - Deserializing from /var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Metadata/Movies/e/81efd610fc02a9443767db5143710b121d125a7.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2023-01-17 13:42:34,482 (7f64a27a0b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/286611/tree'
2023-01-17 13:42:34,492 (7f64a27a0b38) :  INFO (__init__:136) - ******UPDATE CALLED*******
2023-01-17 13:42:34,492 (7f64a27a0b38) :  INFO (__init__:140) - SiteNum: 1071
2023-01-17 13:42:34,492 (7f64a27a0b38) :  INFO (__init__:145) - Provider: siteData18Scenes
2023-01-17 13:42:34,495 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/scenes/3120614"
2023-01-17 13:42:34,739 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "295e7f46d192417492a796b838033534.gz"
2023-01-17 13:42:34,748 (7f64a27a0b38) :  INFO (siteData18Scenes:204) - date: [<Element span at 0x7f64a16d72f0>, <Element span at 0x7f64a309d8f0>]
2023-01-17 13:42:34,748 (7f64a27a0b38) :  INFO (siteData18Scenes:207) - date: Release date: November, 2022  * Movie Release
2023-01-17 13:42:34,748 (7f64a27a0b38) :  INFO (siteData18Scenes:212) - date: u'November, 2022'
2023-01-17 13:42:34,762 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/sys/media_photos.php?s=1&scene=120614&pic=1401"
2023-01-17 13:42:34,921 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "803e389250ac4a558f69f9fa73375b8d.gz"
2023-01-17 13:42:34,924 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/sys/media_photos.php?s=1&scene=120614&pic=3001"
2023-01-17 13:42:35,123 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "cecb6a34ce6e423d9b7e90244d7bcc76.gz"
2023-01-17 13:42:35,127 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.data18.com/sys/media_photos.php?s=1&scene=120614&pic=1201"
2023-01-17 13:42:35,309 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "fdbb0834f4e9452e828b0b23b2893143.gz"
2023-01-17 13:42:35,312 (7f64a27a0b38) :  INFO (siteData18Scenes:280) - Artwork found: 26
2023-01-17 13:42:35,314 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://i1.dt18.com/3/47/120614/0.jpg"
2023-01-17 13:42:35,461 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "164a03212b064e1ab3bebc2f8855809e.gz"
2023-01-17 13:42:35,462 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/01.jpg in posters collection
2023-01-17 13:42:35,462 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/01.jpg in posters collection
2023-01-17 13:42:35,462 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/02.jpg in posters collection
2023-01-17 13:42:35,463 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/03.jpg in posters collection
2023-01-17 13:42:35,463 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/04.jpg in posters collection
2023-01-17 13:42:35,463 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/05.jpg in posters collection
2023-01-17 13:42:35,463 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/06.jpg in posters collection
2023-01-17 13:42:35,464 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/07.jpg in posters collection
2023-01-17 13:42:35,464 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/08.jpg in posters collection
2023-01-17 13:42:35,464 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/09.jpg in posters collection
2023-01-17 13:42:35,464 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/10.jpg in posters collection
2023-01-17 13:42:35,465 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://i1.dt18.com/3/47/120614/11.jpg in posters collection
2023-01-17 13:42:35,465 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/01.jpg in posters collection
2023-01-17 13:42:35,465 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/01.jpg in posters collection
2023-01-17 13:42:35,465 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/02.jpg in posters collection
2023-01-17 13:42:35,466 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/03.jpg in posters collection
2023-01-17 13:42:35,466 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/04.jpg in posters collection
2023-01-17 13:42:35,466 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/05.jpg in posters collection
2023-01-17 13:42:35,466 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/06.jpg in posters collection
2023-01-17 13:42:35,467 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/07.jpg in posters collection
2023-01-17 13:42:35,467 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/08.jpg in posters collection
2023-01-17 13:42:35,467 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/09.jpg in posters collection
2023-01-17 13:42:35,467 (7f64a27a0b38) :  INFO (PAsearchSites:132) - Found https://cdn-v3.dt18.com/img/1/3/120614/10.jpg in posters collection
2023-01-17 13:42:35,468 (7f64a27a0b38) :  INFO (PAsearchSites:137) - Found https://cdn-v3.dt18.com/img/1/3/120614/11.jpg in art collection
2023-01-17 13:42:35,468 (7f64a27a0b38) :  INFO (PAsearchSites:137) - Found https://cdn.dt18.com/media/t/1/scenes/3/5/120614-sofia-lee-stanley-johnson-alpine-lovers.jpg in art collection
2023-01-17 13:42:35,469 (7f64a27a0b38) :  INFO (__init__:149) - Genres
2023-01-17 13:42:35,472 (7f64a27a0b38) :  INFO (__init__:154) - Actors
2023-01-17 13:42:35,475 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.freeones.com/babes?q=Sofia%20Lee"
2023-01-17 13:42:36,105 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "3914259fe11743a1945e577edbb7cf37.gz"
2023-01-17 13:42:36,130 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.freeones.com/sofia-lee/bio"
2023-01-17 13:42:36,934 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "fc0cd370a77c4e6883d5fd67d803e235.gz"
2023-01-17 13:42:36,976 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "http://www.iafd.com/results.asp?searchtype=comprehensive&searchstring=Sofia%20Lee"
2023-01-17 13:42:39,553 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "cb8523a0113f41be968e2bd0cda7eec3.gz"
2023-01-17 13:42:39,560 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "http://www.iafd.com/person.rme/perfid=sofialee/gender=f/sofia-lee.htm"
2023-01-17 13:42:40,119 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "9bd93dc5a5134031a1827c525edae339.gz"
2023-01-17 13:42:40,131 (7f64a27a0b38) :  INFO (PAactors:139) - Sofia Lee found in IAFD 
2023-01-17 13:42:40,132 (7f64a27a0b38) :  INFO (PAactors:140) - PhotoURL: https://www.iafd.com/graphics/headshots/sofialee_f_0014.jpg
2023-01-17 13:42:40,132 (7f64a27a0b38) :  INFO (PAactors:92) - Gender: female
2023-01-17 13:42:40,133 (7f64a27a0b38) :  INFO (PAactors:97) - Actor: Sofia Lee https://www.iafd.com/graphics/headshots/sofialee_f_0014.jpg
2023-01-17 13:42:40,136 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "https://www.freeones.com/babes?q=Stanley%20Johnson"
2023-01-17 13:42:40,786 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "be2d8b33e3074c2c9fc99aceb75e90fa.gz"
2023-01-17 13:42:40,810 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "http://www.iafd.com/results.asp?searchtype=comprehensive&searchstring=Stanley%20Johnson"
2023-01-17 13:42:44,402 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "d33da9ab17434c67b14095414187127c.gz"
2023-01-17 13:42:44,408 (7f64a27a0b38) :  INFO (PAutils:182) - Requesting GET "http://www.iafd.com/person.rme/perfid=stanleyjohnson/gender=m/stanley-johnson.htm"
2023-01-17 13:42:46,079 (7f64a27a0b38) :  INFO (PAutils:281) - GZip request saved as "5e155ec31c1e4087919f0d40cd5e7b83.gz"
2023-01-17 13:42:46,097 (7f64a27a0b38) :  INFO (PAactors:139) - Stanley Johnson found in IAFD 
2023-01-17 13:42:46,097 (7f64a27a0b38) :  INFO (PAactors:140) - PhotoURL: https://www.iafd.com/graphics/headshots/stanleyjohnson_m_4.jpg
2023-01-17 13:42:46,098 (7f64a27a0b38) :  INFO (PAactors:92) - Gender: male
2023-01-17 13:42:46,098 (7f64a27a0b38) :  INFO (PAactors:97) - Actor: Stanley Johnson https://www.iafd.com/graphics/headshots/stanleyjohnson_m_4.jpg
2023-01-17 13:42:46,128 (7f64a27a0b38) :  DEBUG (model:229) - Serializing to /var/packages/PlexMediaServer/shares/PlexMediaServer/AppData/Plex Media Server/Metadata/Movies/e/81efd610fc02a9443767db5143710b121d125a7.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2023-01-17 13:42:46,131 (7f64a27a0b38) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2023-01-17 13:42:46,131 (7f64a27a0b38) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes

```

**Comment by diamondpete** [comment]
@chempke Thanks I think I see the issue. I should have a fix later today
